csharp/src/Ice/PluginInitializationContext.cs

```diff
@@ -4,7 +4,7 @@ namespace ZeroC.Ice
 {
     /// <summary>The plug-in initialization context is passed to plug-ins during initialization allowing plug-ins to
     /// set the Communicator's logger, or add dispatch and invocation interceptors.</summary>
-    public ref struct PluginInitializationContext
+    public sealed class PluginInitializationContext
```

Comment:
Should we make this IDisposable and dispose it after plug-in initialization completes? We don't want users to keep a reference and call AddDispatchInterceptor, AddInvocationInterceptor or set the Logger after initialization completes

Reply:
Yes, fixed.